src/components/ReportActionItem/MoneyRequestPreview/MoneyRequestPreviewContent.tsx

```diff
@@ -427,6 +430,43 @@ function MoneyRequestPreviewContent({
                                             </View>
                                         )}
                                     </View>
+                                    {shouldShowCategoryOrTag && <View style={[styles.threadDividerLine, styles.ml0, styles.mr0, styles.mt1]} />}
+                                    {shouldShowCategoryOrTag && (
+                                        <View style={[styles.flexRow, styles.pt1, styles.alignItemsCenter]}>
+                                            {!!category && (
+                                                <View style={[styles.flexRow, styles.alignItemsCenter, styles.gap1, tag && styles.mw50, tag && styles.pr1, styles.flexShrink1]}>
+                                                    <Icon
+                                                        src={Expensicons.Folder}
+                                                        height={variables.iconSizeExtraSmall}
+                                                        width={variables.iconSizeExtraSmall}
+                                                        fill={theme.icon}
+                                                    />
+                                                    <Text
+                                                        numberOfLines={1}
+                                                        style={[styles.textMicroSupporting, styles.pre, styles.flexShrink1]}
+                                                    >
+                                                        {category}
+                                                    </Text>
+                                                </View>
+                                            )}
+                                            {!!tag && (
```

Comment:
coming from this issue https://github.com/Expensify/App/issues/53454, here we’re displaying the tag name if tag value is available. However, this alone is not sufficient for display in the preview. Rooms created under a workspace with dependent tags enabled will also show the tags in the preview. That said, tags should only appear in the expense preview if the report is a policy expense chat. More details can be found here: https://github.com/Expensify/App/issues/53454#issuecomment-2514595733